public/app/features/dashboard-scene/utils/dashboardSceneGraph.ts

```diff
@@ -17,6 +17,18 @@ function getTimePicker(scene: DashboardScene) {
   return null;
 }
 
+function getRefreshPicker(scene: DashboardScene) {
```

Comment:
Please add test for that. Without the test we can easily introduce a regression if we made a decision to change where in the hierarchy the refresh picker is.